src/components/DeleteAllQuestionsToggle.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-env browser */
```

Comment:
This shouldn't be necessary here - was only necessary in the notifications toggle because we used `window` there.